apps/_default/__init__.py

```diff
@@ -1,7 +1,8 @@
-from py4web import action, __version__
+from py4web import action, __version__, redirect, URL
 
 
 @action("index")
 @action.uses("index.html")
 def index():
+    redirect(URL('OpenGenomeExplorer'))
```

Comment:
just a convenience thing I've been doing, redirects the default py4web page to our project page so you just go to `localhost` instead of `localhost/OpenGenomeProject`